blockchain/blockdao.go

```diff
@@ -190,34 +164,6 @@ func (dao *blockDAO) initStores() error {
 	dao.topIndex.Store(maxN)
 	return nil
 }
-func (dao *blockDAO) countActions() error {
```

Comment:
if this function deleted,GetChainMeta api cannot get total actions when -plugin=gateway is not enabled or cfg.Chain.EnableAsyncIndexWrite is false